packages/invoices/src/InboundShipment/ListView/AppBarButtons.tsx

```diff
@@ -8,13 +9,14 @@ import {
   ButtonWithIcon,
   Grid,
   useTranslation,
+  useToggle,
```

Comment:
I also moved this modal in a couple of places to the `AppBarButtons` component as that's the lowest it needs to be - it doesn't actually need to be in the `ListView` which is essentially the root - not ideally for triggering state updates (though it didn't actually cause any issues, I suppose)